src/input/EasiMeshSize.h

```diff
@@ -1,17 +1,14 @@
 #ifndef PUMGEN_EASIMESHSIZE_H
 #define PUMGEN_EASIMESHSIZE_H
 
-#include "easi/Component.h"
+#include <easi/YAMLParser.h>
+#include <easi/Component.h>
```

Comment:
These should be " not <> - the behavior is different actually. Only with " you get the correct search path. (This actually lead to a problem for me once, so it's not just nitpicking)

Reply:
ha ok, then I misunderstood your previous link. I thought as easi includes are not in the project (e.g. in ~/include) they have to be included with <>

Reply:
Actually, I think I have confused myself - angular brackets are the correct choice here - I'll fix it, sorry.

In principle, <> for everything external, "" for everything internal. And easi is obviously external.